.github/workflows/lint.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/setup-go@v4
         name: Set up Go 1.x
         with:
-          go-version: "1.21.9"
+          go-version: "1.22.7"
```

Comment:
You could use `go-version-file: 'path/to/go.mod'` instead. It's one less thing to remember to update in a future go version update.

We use it [here](https://github.com/project-koku/koku-metrics-operator/blob/main/.github/workflows/ci.yaml#L20-L23).

Reply:
Thanks for that